src/packages/ganache/webpack/webpack.node.config.ts

```diff
@@ -51,6 +53,11 @@ const config: webpack.Configuration = merge({}, base, {
   externals: [
     //#region dependencies that have the potential to compile something at install time
     "@trufflesuite/bigint-buffer",
+    "@trufflesuite/uws-js-unofficial",
+    "abstract-leveldown",
+    "async-eventemitter",
+    "leveldown",
+    "levelup",
```

Comment:
Should these fall under the `things api-extractor can't handle, so we don't bundle them` section?

Reply:
I've moved `abstract-leveldown` and `async-eventemitter`. `levelup` is being removed and `leveldown` should remain there (it's already there in develop)